bundle/src/main/java/com/adobe/acs/commons/assets/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2018 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+@org.osgi.annotation.versioning.Version("4.1.0")
```

Comment:
hi @davidjgonzalez, this is because of Baseline analysis:
[ERROR] com.adobe.acs.commons.assets: Version increase required; detected 1.0.0, suggested 4.1.0